WORKSPACE

```diff
@@ -33,12 +33,22 @@ node_repositories(package_json = ["//:package.json"])
 
 http_archive(
     name = "io_bazel_rules_go",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.7.0/rules_go-0.7.0.tar.gz",
     sha256 = "91fca9cf860a1476abdc185a5f675b641b60d3acf0596679a27b580af60bf19c",
+    url = "https://github.com/bazelbuild/rules_go/releases/download/0.7.0/rules_go-0.7.0.tar.gz",
```

Comment:
nit: Consider reverting this part

Reply:
FYI I have a PR out that upgrades this to 0.7.1